src/registrar/tests/test_views.py

```diff
@@ -1219,6 +1219,8 @@ def setUp(self):
         self.app.set_user(self.user.username)
         self.client.force_login(self.user)
 
+
+class TestDomainDetail(TestDomainOverview):
```

Comment:
**question (non-blocking)**: Same question about duplicate name changes in lots of PRs?

Reply:
TestDomainOverview tests were being run multiple times.  By creating new class, TestDomainDetail, I cut the processing time of the whole suite by maybe 40 seconds
Also, for some reason, some of the iterations of the tests were failing.  Running them only once, they pass.  Probably something missing in some SetUp or TearDown.